pyproj/crs.py

```diff
@@ -49,10 +50,9 @@
 from pyproj.geod import Geod
 
 
-def _dict2string(projparams):
```

Comment:
I found dict2string a clearer name, as it that what it does (it does not return a CRS from a dict). I was actually confused by this, while reading the `_from_string` method below, thinking that it returned a CRS in case of the json string.